### DISCUSSION TITLE
How to get correct cursor color with Catppuccin?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [neovide/neovide#2891] [Q&A]
How it looks in Kitty:
<img width="780" alt="CleanShot 2024-12-12 at 07 50 08@2x" src="https://github.com/user-attachments/assets/1c281747-5057-456e-8cb3-46a0c994b881" />

How it looks in Neovide:
<img width="998" alt="CleanShot 2024-12-12 at 07 50 30@2x" src="https://github.com/user-attachments/assets/c28a5c2f-9dda-4b3a-a3a3-ec5741e7946a" />

I filed a bug report here: https://github.com/catppuccin/nvim/issues/810. Here's the repro.lua:

```lua
-- LazyNvim Setup
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
vim.opt.clipboard = "unnamedplus"
if not (vim.uv or vim.loop).fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "https://github.com/folke/lazy.nvim.git",
    "--branch=stable", -- latest stable release
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)
require("lazy").setup({
  { 
		"catppuccin/nvim", 
		name = "catppuccin", 
		priority = 1000,
	},
})

require("catppuccin").setup({
    term_colors = true,
})

vim.cmd.colorscheme "catppuccin-latte"
```

**Comment by theherk** [comment]
I use this function, [set_cursor_colors](https://github.com/theherk/commons/blob/72f00ad1923c505ce5d036671f460e8b4b3fd8b5/.config/nvim/lua/config/util.lua#L210-L213) then call it [like this](https://github.com/theherk/commons/blob/72f00ad1923c505ce5d036671f460e8b4b3fd8b5/.config/nvim/lua/config/neovide.lua#L11-L16).
